app/src/main/java/com/github/palFinderTeam/palfinder/profile/FirebaseProfileService.kt

```diff
@@ -55,20 +55,22 @@ open class FirebaseProfileService @Inject constructor(
                 db.collection(PROFILE_COLL).document(user.uuid),
                 FOLLOWING_PROFILES, FieldValue.arrayUnion(targetId)
             )
-            if (updateAchievementsFollower(user) != null) {
+            val updatedUser = user.copy(following = user.following.plus("newUser"))
```

Comment:
Why is there an hardcorded "newUser" string here?

Reply:
that is meant to add a +1 to the size of the follower/following list, in order to calculate achievements properly without re-fetching the user